target/linux/ipq40xx/image/Makefile

```diff
@@ -387,6 +387,20 @@ define Device/glinet_gl-b1300
 endef
 TARGET_DEVICES += glinet_gl-b1300
 
+define Device/glinet_gl-s1300
+	$(call Device/FitImage)
+	DEVICE_VENDOR := GL.iNet
+	DEVICE_MODEL := GL-S1300
+	BOARD_NAME := gl-s1300
```

Comment:
Is this really needed?

Reply:
I found if delete these lines, the menuconfig won't show the target profile, so it's needed.

Reply:
I was referring to the BOARD_NAME only. The DEVICE_* stuff is indeed needed for menuconfig.

Reply:
Thanks. I check BOARD_NAME is only included in sysupgrade.meta for variable "board", and that's not used for sysupgrade, so remove it is okay as you mention.

Reply:
So, remove it?